src/main/java/uk/gov/hmcts/reform/orgrolemapping/controller/RefreshController.java

```diff
@@ -58,4 +67,34 @@ public RefreshController(RefreshOrchestrator refreshOrchestrator) {
         return refreshOrchestrator.refresh(jobId, userRequest);
 
     }
+
+
+    @PostMapping(
+            path = "/am/role-mapping/judicial/refresh",
+            produces = V1.MediaType.MAP_ASSIGNMENTS,
```

Comment:
we can create a new entry for this endpoint like REFRESH_JUDICIAL_ASSIGNMENTS in the MediaType.